onnxruntime/test/python/quantization/test_quantize_static.py

```diff
@@ -98,6 +98,19 @@ def test_static_quant_config(self):
         check_model_correctness(self, self._model_fp32_path, quant_model_path, data_reader.get_next())
         data_reader.rewind()
 
+    def test_smooth_quant(self):
```

Comment:
I suggest to add a check that when neural-compressor is not installed, we can skip the test. In this way, we need not install neural-compressor and run this test in every pipeline.

Reply:
Good point. Thanks for the suggestions.

Reply:
I have updated my PR according to your suggestions, thanks a lot 😄